tests/performance/test_performance.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+export PYTHONPATH=/home/kbyers/netmiko/tests
+cd /home/vera/netmiko/tests/performance
+
+pip install pyaml pygal jinja2
```

Comment:
Let's not do the pip install and instead do a "pip show" on each of the packages and print a message if they are not installed. I say this as the most likely case is probably me trying to run this using the wrong virtual environment and I would rather know that I was in the wrong venv than installing the packages.